agents/mox_dynamics_crm/migrations/mongo_interface.py

```diff
@@ -5,20 +5,20 @@
 
 
 # Configuration
-config = get_config()
-mongo = config["mongodb"]
+config = get_config("mongodb")
 
 # Create mongo client connection pool
 client = MongoClient(
-    host=mongo["db_host"],
-    port=int(mongo["db_port"]),
-    authSource=mongo["db_name"],
-    username=mongo["db_user"],
-    password=mongo["db_pass"]
+    host=config["db_host"] or None,
```

Comment:
Denne her "or None" opstår kun, hvis `db_host`er sat til en tom streng i konfigurationen. Vil det ikke få det hele til at fejle?

Reply:
For i så fald kunne man måske have et check på det og stoppe, hvis de nødvendige parametre ikke er der.

Reply:
Jeg er enig i at det er pænere at stoppe applikationen - connect metoden falder tilbage på "localhost" hvis host=None, hvilket er grunden til at det er blevet tilføjet. 

(Migrations funktionaliteten bliver dog ikke brugt fremover tænker jeg - det er mere for historikkens skyld jeg har tilføjet det)

Reply:
Yes, det er nok fint.